client/electron/electron-builder.json

```diff
@@ -0,0 +1,52 @@
+{
+  "artifactName": "Outline-Client.${ext}",
+  "asarUnpack": [ "client" ],
+  "directories": {
+    "buildResources": "output/client/electron",
+    "output": "output/client/electron/build"
+  },
+  "extraMetadata": {
+    "main": "output/client/electron/index.js"
+  },
+  "files": [
```

Comment:
In the future, I'd like to see all of this under a single directory. Even if we need to copy them.